### PR TITLE
Widespace bid adapter: support for the floors module

### PR DESCRIPTION
**Pull request by talha08**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [x] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Widespace adapter: add support for adomain and floors module for Prebid 5.
#6465
#6650

### CODE REVIEW

**Comment by patmmccann** [comment]
tyvm!

**Comment by talha08** [comment]
I can see some test cases failed in other adapters.  But here it shows, **Your tests failed on CircleCI**

I'm new here, What should I need to do? Please let me know!
@patmmccann

File: modules/widespaceBidAdapter.js [hunk 1]
```diff
@@ -256,4 +252,21 @@ function getTopWindowReferrer() {
...
+const getBidFloor = (bid) => {
+  if (!utils.isFn(bid.getFloor)) {
+    return bid.params.bidfloor ? bid.params.bidfloor : null;
+  }
+
+  let bidFloor = bid.getFloor({
+    mediaType: '*',
+    size: '*',
+    currency: 'EUR'
+  });
```
Comment (Fawke) [inline modules/widespaceBidAdapter.js:264]: This can be wrapped in a try/catch block incase getFloor throws an error.

**Review by Fawke — CHANGES_REQUESTED** [review]
Hi @talha08,

Thanks for this PR. Change looks good but I had a few things which can be looked at. 
1. I don't see any unit test case covering the new `getBidFloor` function that you wrote.
2. You need to submit a docs PR to change this field, `Floors Module Support` to `yes`.

Do not worry about the test cases for other adapters failing on cirle ci, I see that they're passing now. Some tests are flaky which can cause the circle ci build to fail.

**Comment by Fawke** [comment]
@talha08 Hey, just checking! I see no updates on this PR for quite a few days, do you need any help with this?

**Comment by Fawke** [comment]
@ChrisHuie Do we wanna close this PR, I haven't seen any activity on this since two months.

**Comment by ChrisHuie** [comment]
@Fawke I will just change the assignee to me so you don't have to worry about it :) I will try and reach out.

**Comment by ChrisHuie** [comment]
@talha08 is this pr still being worked on? Haven't heard anything in a while and we are cleaning out stale prs. If I don't hear back by Thanksgiving I will just go ahead and close this out and you can reopen again later if needed.

**Pull request closed without merging**